contracts/fuses/stake_dao_v2/StakeDaoV2SubstrateLib.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.26;
+
+import {PlasmaVaultConfigLib} from "../../libraries/PlasmaVaultConfigLib.sol";
+
+enum StakeDaoV2SubstrateType {
+    UNDEFINED,
+    RewardVault,
+    ExtraRewardToken
```

Comment:
Do we need this in the substrates? 
It is only passed as a parameter during the claim, but we do not perform any actions on these tokens, so there is no risk that they will negatively affect the vault.

Reply:
Ok, we can do in this way that if substrate is not used as a contract with any method execution inside fuse then don't have to be in substrates